server/stores/config.py

```diff
@@ -14,6 +14,7 @@ class Config:
     def __init__(self, project: Project):
         self.system = project.system / "config.json"
         self.project = project.private / "config.json"
+        self.hideWelcomeScreen = "config.json"
```

Comment:
does this make any sense @roll ?

Reply:
@guergana 
You need to add this config to `SystemConfig` (the one that is stored in the user config dir):

```
class SystemConfig(BaseModel):
    openaiApiKey: str | None = None
    hideWelcomScreen: bool | None = None # or false by default
 ```